python/paddle/distributed/communication/utils.py

```diff
@@ -0,0 +1,27 @@
+#   Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Don't use the name utils. Use `xxx_utils` because we can't estimate the size of this file.